File: app/src/main/java/ru/ushell/app/api/body/BodyRequestSingIn.java
package ru.ushell.app.api.body;

public class BodyRequestSingIn {
    private String email;
    private String password;

    public String getEmail() {
        return email;
    }

    public BodyRequestSingIn setEmail(String email) {
        this.email = email;
        return this;
    }

    public String getPassword() {
        return password;
    }

    public BodyRequestSingIn setPassword(String password) {
        this.password = password;
        return this;
    }

}

File: app/src/main/java/ru/ushell/app/api/body/BodyRequestSingIn.kt
package ru.ushell.app.api.body

class BodyRequestSingIn {
    private var email: String? = null
    private var password: String? = null

    fun setEmail(email: String?): BodyRequestSingIn {
        this.email = email
        return this
    }

    fun setPassword(password: String?): BodyRequestSingIn {
        this.password = password
        return this
    }
}
